x-pack/plugins/cases/server/routes/api/cases/comments/get_all_comment.ts

```diff
@@ -14,20 +14,25 @@ import { flattenCommentSavedObjects, wrapError } from '../../utils';
 import { CASE_COMMENTS_URL } from '../../../../../common/constants';
 import { defaultSortField } from '../../../../common';
 
-export function initGetAllCommentsApi({ caseService, router, logger }: RouteDeps) {
+export function initGetAllCommentsApi({ caseService, router, logger, subCasesEnabled }: RouteDeps) {
+  const querySchema = subCasesEnabled
```

Comment:
It seems that this pattern is being applied in our routes. Why we need to remove this parameters?

Reply:
I removed the flag from being passed around just import the global directly now.